tests/integration/loki-tester/lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
Was this committed by mistake? Should probably be copied into as part of the test.